src/databricks/labs/ucx/cli.py

```diff
@@ -646,6 +646,17 @@ def migrate_tables(
         workspace_contexts = _get_workspace_contexts(w, a, run_as_collection)
     for workspace_context in workspace_contexts:
         deployed_workflows = workspace_context.deployed_workflows
+
+        try:
+            workspace_context.verify_progress_tracking.verify()
```

Comment:
@FastLee : A rename would wake sense as this is not the place to "verify progress tracking". Either scope it to check if the assessment has ran (like the warning says) or rename the class to something broader than "verify progress tracking"